src/rpc/cli.cpp

```diff
@@ -103,6 +103,11 @@ void cli::run()
          }
          else
             std::cout << itr->second( result, args ) << "\n";
+
+         if (method == "quit") 
+         {
+            stop();
```

Comment:
Why not `break`?

Reply:
If this would work, I think it should be placed before line 98.

Reply:
Why ```break``` and not ```stop``` ? Is ```cli::stop``` wrong or bad ?
I've placed this after line 98 because ```cli_wallet::quit``` command can run its own method before stopping completely, so should I remove ```quit``` command from wallet.cpp so in help info users won't see this command description and should I place this line before 98 ?

Reply:
I asked because some lines above there is a `break`, when got an EOF.

> I've placed this after line 98 because cli_wallet::quit command can run its own method before stopping completely

Makes sense. Although your code would work, I still think it's better to quit based on returned value but not the hard-coded command "quit".

Reply:
@abitmore yes you're absolutely right about hard-coded "quit" ! I thought about it too and now I think how to make it correctly with the best quality ! Thank you very much for the suggestion and your time !

Reply:
changes were made